runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -94,6 +94,7 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
    bool isValueTypeClass(TR_OpaqueClassBlock *);
    bool isValueTypeClassFlattened(TR_OpaqueClassBlock *clazz);
    bool isValueBasedOrValueTypeClass(TR_OpaqueClassBlock *);
+   bool isClassHasIdentity(TR_OpaqueClassBlock *);
```

Comment:
Could we add some Doxygen documentation here?

Reply:
Added Doxygen documentation in 3c2a2b4